lib/terminus.spec.js

```diff
@@ -136,16 +136,91 @@ describe('Terminus', () => {
     })
 
     it('returns 503 once signal received', (done) => {
-      execFile('node', ['lib/standalone-tests/terminus.onsignal.fail.js'])
+      let responseAssertionsComplete = false
+
+      // We're only truly finished when the response has been analyzed and the forked http process has exited,
+      // freeing up port 8000 for future tests
+      const finished = () => {
```

Comment:
do we need the finished function? or can we run these in the execFile callback?

Reply:
Ah good catch - this particular finished() function is unnecessary